Cargo.toml

```diff
@@ -35,30 +35,29 @@ anyhow = "1.0"
 serde = { version = "1.0.103", default-features = false, features = ["derive"] }
 tokio = { version = "1.4", features = ["full"] }
 
-cw-orch = { path = "./cw-orch", version = "0.23.0" }
-cw-orch-daemon = { path = "./cw-orch-daemon", version = "0.23.2" }
-cw-orch-core = { path = "packages/cw-orch-core", version = "1.1.1" }
```

Comment:
I think because we added a "deprecated" flag, we should probably do a minor bump vs. a patch dump on core.

Reply:
Already published with patch as we discussed yesterday. I think it's fine, as if doesn't break any existent code compilation, only few warnings